images/win/Windows2019-Azure.json

```diff
@@ -591,8 +591,8 @@
         {
             "type": "powershell",
             "environment_vars": [
-                "GO_VERSIONS=1.10.8,1.11.12,1.12.7,1.13",
-                "GO_DEFAULT=1.12.7"
+                "GO_VERSIONS=1.10,1.11,1.12,1.13",
```

Comment:
Could you please add spaces between versions for readability?
`"GO_VERSIONS=1.10, 1.11, 1.12, 1.13",`
We can trim lines later

Reply:
Done.